### DISCUSSION TITLE
Autoscroll with timer

### DISCUSSION THREAD

**Discussion by zubkovla** [sumatrapdfreader/sumatrapdf#3175] [Ideas]
Fantastic feature would be an option to set up automatic page scrolling in seconds, lets say 15 seconds.
But, even more awesome feature would be to tie this value to word (or character) number on a page, so, say you read a page, when you finish reading and move to the next one, you press autoscroll, and it gives you exactly the amount of time per word or character, which you've spent on the previous page.
Bongo.

**Comment by zubkovla** [comment]
Some text or concepts may be presented on a page in images, which also takes time to consume, so images should probably be also taken into account when considering time delay

**Reply by GitHubRulesOK** [reply]
hardware auto scroll is user variable, move slightly more from start is faster less is slower, but to change page behaviour rate is also highly user subjective.

To set a page timer you can add an external (or added internal) command
Never needed such, but should be possible to export current page and start the external timer to change to next page however that would not know when to stop, so have never attempted such. (doable via cmd pdfinfo or mutool, but niche)

**Comment by Sensor66** [comment]
I second @zubkovla's idea. 
My daily problem is that scrolling --at the slowest speed-- is just a bit too fast ... if it is simpler, please give an option to lower the minimum speed. 
Thanks for the great reader!
